modules/persistent-storage-rhv.adoc

```diff
@@ -0,0 +1,72 @@
+[id="persistent-storage-rhv_{context}"]
+= Creating a persistent volume on RHV
+
+When you create a `PersistentVolumeClaim` (PVC) object, {product-title} provisions a new persistent volume (PV) and creates a `PersistentVolume` object.
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* You provided the correct {rh-virtualization} credentials in  `ovirt-credentials` secret.
+* You have installed the oVirt CSI Driver.
+* You have defined at least one storage class.
+
+.Procedure (UI)
```

Comment:
So this is how I included both types of procedures in a single module: https://docs.openshift.com/container-platform/4.6/networking/hardware_networks/installing-sriov-operator.html

Another approach I've seen is to split the procedure into two branches:

* If you are using the CLI...
  1. abc
  2. xyz
* If you are using the web console...
  1. def

But we don't really ever use headers such as: _Procedure (UI)_